src/schema/sources.ts

```diff
@@ -270,6 +275,21 @@ export const typeDefs = /* GraphQL */ `
       first: Int
     ): SourceMemberConnection! @auth
 
+    """
+    Check whether user has membership access to a squad
+    """
+    checkUserMembership(
+      """
+      The source to check whether the user is a part of
+      """
+      sourceId: ID!
+
+      """
+      Username or the user id to check
```

Comment:
Just for clarity this only checks on id no?

Reply:
Correct. This should be just ID. Let me refactor.